utils/rnaview.py

```diff
@@ -0,0 +1,187 @@
+    lines = output.splitlines()
+    
+    # Step 1: Collect all pairings
+    for line in lines:
+        if line.startswith('pair'):
+            parts = line.split()
+            if len(parts) >= 5:
+                pos1 = int(parts[2]) - 1  # Convert to 0-based index
+                pos2 = int(parts[3]) - 1  # Convert to 0-based index
+                
+                if pos1 < len(sequence) and pos2 < len(sequence):
+                    pair = sequence[pos1] + sequence[pos2]
+                    if pair in canonical_pairs:
+                        # Initialize sets if not already
+                        if pos1 not in pairings:
+                            pairings[pos1] = set()
+                        if pos2 not in pairings:
+                            pairings[pos2] = set()
+                        
+                        # Add the pairings
+                        pairings[pos1].add(pos2)
+                        pairings[pos2].add(pos1)
+    
+    # Step 2: Identify bases that pair with more than one base
+    multi_paired_bases = {pos for pos, partners in pairings.items() if len(partners) > 1}
+    
+    # Step 3: Construct the dot-bracket notation
+    for pos1, partners in pairings.items():
+        # Skip if base is multi-paired
+        if pos1 in multi_paired_bases:
+            continue
+        # Get the single partner
+        pos2 = next(iter(partners))
+        # Skip if partner is multi-paired or already assigned
+        if pos2 in multi_paired_bases or dot_bracket[pos1] != '.' or dot_bracket[pos2] != '.':
+            continue
+        if pos1 < pos2:
+            dot_bracket[pos1] = '('
+            dot_bracket[pos2] = ')'
+        else:
+            dot_bracket[pos2] = '('
+            dot_bracket[pos1] = ')'
+    
+    # Output the dot-bracket notation
+    dot_bracket_str = ''.join(dot_bracket)
+    print(dot_bracket_str)
+    return dot_bracket_str if dot_bracket else None
+
+
+def _extract_sequence(pdb_file):
+    with open(pdb_file, 'r') as file:
+        for record in SeqIO.parse(file, 'pdb-atom'):
+            return str(record.seq)
+    return None
+
+def write_fasta(pdb_file, sequence, dot_bracket):
+    fasta_filename = pdb_file.replace(".pdb", ".fasta")
+    base_filename = os.path.basename(pdb_file).replace(".pdb", "")
+    with open(fasta_filename, 'w') as fasta_file:
+        fasta_file.write(f">{base_filename}\n")
+        fasta_file.write(f"{sequence}\n")
+        fasta_file.write(f"{dot_bracket}\n")
+
+def run_r2dt(fasta_file):
+    fasta_path = Path(fasta_file)
+    input_folder = fasta_path.parent
+    filename_stem = fasta_path.stem
+    r2dt_dir = input_folder / 'r2dt'
+    
+    # Check if the directory exists, if not, create it
+    if not r2dt_dir.exists():
+        r2dt_dir.mkdir(parents=True)
+    
+    try:
+        output_path = r2dt_dir / filename_stem
+        result = subprocess.run(['r2dt.py', 'templatefree', str(fasta_path), str(output_path)], capture_output=True, text=True)
```

Comment:
Could you please add the `--quiet` option to the R2DT command so that it does not print any output?